examples/gmd_aquaplanet.py

```diff
@@ -0,0 +1,113 @@
+import climt
+from sympl import (
+    PlotFunctionMonitor, NetCDFMonitor,
+    TimeDifferencingWrapper, UpdateFrequencyWrapper,
+    DataArray
+)
+import numpy as np
+from datetime import timedelta
+
+
+def plot_function(fig, state):
+
+    ax = fig.add_subplot(2, 2, 1)
+    state['specific_humidity'].mean(
+        dim='longitude').plot.contourf(
+            ax=ax, levels=16, robust=True)
+    ax.set_title('Specific Humidity')
+
+    ax = fig.add_subplot(2, 2, 3)
+    state['eastward_wind'].mean(dim='longitude').plot.contourf(
+        ax=ax, levels=16, robust=True)
+    ax.set_title('Zonal Wind')
+
+    ax = fig.add_subplot(2, 2, 2)
+    state['air_temperature_tendency_from_convection'].transpose().mean(
+        dim='longitude').plot.contourf(
+        ax=ax, levels=16, robust=True)
+    ax.set_title('Conv. Heating Rate')
+
+    ax = fig.add_subplot(2, 2, 4)
+    state['air_temperature'].mean(dim='longitude').plot.contourf(
+        ax=ax, levels=16)
+    ax.set_title('Temperature')
+
+    fig.tight_layout()
+
+
+fields_to_store = ['air_temperature', 'air_pressure', 'eastward_wind',
+                   'northward_wind', 'air_pressure_on_interface_levels',
+                   'surface_pressure', 'upwelling_longwave_flux_in_air',
+                   'specific_humidity', 'surface_temperature',
+                   'convective_heating_rate', 'latitude', 'longitude']
+# Create plotting object
+monitor = PlotFunctionMonitor(plot_function)
+netcdf_monitor = NetCDFMonitor('gcm_without_seasonal_cycle.nc',
+                               write_on_store=True,
+                               store_names=fields_to_store)
+
+climt.set_constants_from_dict({
```

Comment:
This should just be `sympl.set_constant('stellar_irradiance', value=200, units'W m^-2')`. It isn't necessary to set them from a dict when we're only setting one!